### PR TITLE
YO30 and YO46

### PR DESCRIPTION
**Pull request by alou-saechao**
- user is able to send request
- user is able to view list of request that are pending
- user is able to approve a request
- user is able to deny a request

### CODE REVIEW

File: app/scripts/app.ts [hunk 1]
```diff
@@ -19,6 +21,10 @@ angular.module('app', ['app.controllers','ngRoute','ngStorage','ui.bootstrap', '
       templateUrl:    '/views/signupView.html',
       controller:     'Signup.Controller'
     });
+    $routeProvider.when('/ApproveDeny', {
+      templateUrl:    '/views/ApproveDenyView.html',
+      controller:     'Response.Controller',
+    });
```
Comment (domshyra) [inline app/scripts/app.ts:34]: May need to add resolve here so that a user cannot directly route to AppoveDeny

**Comment by ladinu** [comment]
#peer

File: app/scripts/components/Request/RequestController.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
+/// <reference path="../../app.ts"/>
+/// <reference path='carpoolIDModel.ts'/>
+
+// send join request
+module Request {
+
+  export interface Scope {
+    sendRequest: Function;
+    carpoolID: any;
```
Comment (justinshuck) [inline app/scripts/components/Request/RequestController.ts]: We might want to change the type of carpoolID here to utilize typescript

**Comment by justinshuck** [comment]
@alou-saechao  We'll need to pull from develop again, but other than that good work!

**Comment by alou-saechao** [comment]
- I have added the resolve feature to the ApproveDeny page.
- Changed type of carpoolID from 'any' to 'string'.
- updated my GET and POST to use ConfigService

**Comment by ianaldrighetti** [comment]
Great work Lou! 

#peer

**Comment by alou-saechao** [comment]
Do we need another peer for this before we merge this into develop? Not sure who does the merge.

**Comment by ianaldrighetti** [comment]
@alou-saechao You just need two. As long as two people have peer reviewed it you can hit the "Merge pull request button."

**Pull request merged**